### PR TITLE
Add an option to remove username 

### LINKED ISSUE #183: Option : Remove usernames from posts
Usernames should not be displayed in posts and all submissions should be anonymous (à la 4chan?)

### PR DESCRIPTION
**Pull request by dattaz**
Fix #183

### CODE REVIEW

File: sotoki/sotoki.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
   --ignoreoldsite                               Ignore Stack Exchange closed sites
   --nopic                                       Doesn't download images
   --no-userprofile                              Doesn't include user profiles
+  --no-username                                 Doesn't include user profiles
```
Comment (rgaudin) [inline sotoki/sotoki.py:27]: That's not what it does

**Review by rgaudin — CHANGES_REQUESTED** [review]
Thanks @dattaz 

In this version, the username is still visible on the user's profile page, which is still both included and linked via the avatar.
The displayName should be removed there as well.

We know that in the Bard context we'd also use `--no-userprofile` and `--nopic` but it's just not consistent.

@kelson42, do we want to to have this feature separated (as it is now) or do we want it to automatically enable --no-user-profile ?

**Comment by kelson42** [comment]
@dattaz @rgaudin I’m not sure what we should do with this PR?

**Comment by rgaudin** [comment]
targets previous code base. we have no choice but to close it

**Pull request closed without merging**